plugin/src/test/groovy/com/novoda/test/TestAndroidKotlinProject.groovy

```diff
@@ -0,0 +1,78 @@
+package com.novoda.test
+
+class TestAndroidKotlinProject extends TestProject<TestAndroidKotlinProject> {
+    private static final Closure<String> TEMPLATE = { TestAndroidKotlinProject project ->
+        """
+buildscript {
+    repositories { 
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.1'  
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.20'
+    }
+}
+plugins {
+    ${formatPlugins(project)}
+}
+repositories { 
+    google()
+    jcenter()
+}
+apply plugin: 'com.android.library'  
+apply plugin: 'kotlin-android'
```

Comment:
This has been added

Reply:
to be honest you can also use the `plugins` and your `withPlugin` method to add these dynamically from test code. 
The plugin id is `org.jetbrains.kotlin.android`
I think also `com.android.library` works too.

Reply:
I actually tried that for the kotlin plugin, but it didn't work. I got this error `Extension with name 'android' does not exist`. 
And afaik this is because of the order. `com.android.library` needs to be applied first and I couldn't find that one [here](https://plugins.gradle.org/search?term=android+library&page=2).